#!/usr/bin/env python

PACKAGE = "hello"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# name, type, bitmask, description, default, min, max
gen.add("greeter_name", str_t, 0, "Name of the person to be greeted","Hodor")
gen.add("greeter_hz", int_t, 0, "Frequency of greeting publications", 10, 0, 30)

# package_name, node_name, generated_files_prefix 
exit(gen.generate(PACKAGE, "hello_config", "Hello"))